src/test/java/integration/TestProperties.java

```diff
@@ -0,0 +1,12 @@
+package integration;
+
+import org.springframework.test.context.DynamicPropertyRegistry;
+
+public class TestProperties {
```

Comment:
@hizmailovich can I suggest a refactoring direction?

```java
class PropertyOf {
private final DynamicRegistry reg;
private final String key;
private final Object value;

@Override
void set() {
this.reg.add(this.key, this.value);
}
}
```
feel free to improve/wrap this code

WDYT?